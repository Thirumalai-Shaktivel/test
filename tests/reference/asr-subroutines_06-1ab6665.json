{
    "basename": "asr-subroutines_06-1ab6665",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/subroutines_06.f90",
    "infile_hash": "a9ec95251f8c93a472cffed55c7a1f91a062feb2cab735a8543c8d38",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-subroutines_06-1ab6665.stdout",
    "stdout_hash": "91ae86f4ebab1c8ca52721d4914312853393974040dfcf54d9e4440b",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}