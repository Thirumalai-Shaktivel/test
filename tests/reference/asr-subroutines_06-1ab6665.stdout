(TranslationUnit (SymbolTable 1 {bitset: (Module (SymbolTable 2 {assign_large: (Subroutine (SymbolTable 3 {set1: (Variable 3 set1 Out () () Default (Integer 4 []) Source Public Required .false.), set2: (Variable 3 set2 In () () Default (Integer 4 []) Source Public Required .false.)}) assign_large [(Var 3 set1) (Var 3 set2)] [] Source Public Interface () .true. .true.), assign_logint8_large: (Subroutine (SymbolTable 4 {logical_vector: (Variable 4 logical_vector In () () Default (Logical 4 [(() ())]) Source Public Required .false.), self: (Variable 4 self Out () () Default (Integer 4 []) Source Public Required .false.)}) assign_logint8_large [(Var 4 self) (Var 4 logical_vector)] [] Source Public Interface () .true. .true.), ~assign: (CustomOperator 2 ~assign [2 assign_large 2 assign_logint8_large] Public)}) bitset [] .false.)}) [])
